common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -74,3 +99,10 @@ steps:
   - script: node common/scripts/install-run-rush.js docs
     displayName: rush docs
     workingDirectory: ${{ parameters.workingDir }}
+
+  # # Will run if even there is a failure somewhere else in the pipeline.
+  # - template: ../templates/publish-test-results.yaml
```

Comment:
The publish test results should be uncommented here as it's good to have that in the core-build.